charts/k0s/templates/statefulset.yaml

```diff
@@ -289,6 +295,10 @@ spec:
         volumeMounts:
           - name: helm-cache
             mountPath: /.cache/helm
+          - name: k0s-binary
+            mountPath: /k0s-binary
+          - name: run-k0s
+            mountPath: /run/k0s
```

Comment:
I guess we will need the k0s-config mount and data mount here instead of the values, also would be good to add the vcluster.extraMounts here